src/PKSim.Presentation/Views/Individuals/IIndividualSettingsView.cs

```diff
@@ -12,13 +12,15 @@ public interface IIndividualSettingsView : IView<IIndividualSettingsPresenter>,
    {
       void BindToSettings(IndividualSettingsDTO individualSettingsDTO);
       void BindToParameters(IndividualSettingsDTO individualSettingsDTO);
+      void BindToDiseaseState(IndividualSettingsDTO individualSettingsDTO);
       void BindToSubPopulation(IEnumerable<CategoryParameterValueVersionDTO> subPopulation);
       void RefreshAllIndividualList();
       bool AgeVisible { get; set; }
       bool HeightAndBMIVisible { get; set; }
       bool IsReadOnly { get; set; }
       bool SpeciesVisible { get; set; }
       bool GestationalAgeVisible { get; set; }
+      bool DiseaseStateVisible { get; set; }
```

Comment:
Following the logic. I would probably do that differntly nowdays but this is good enough